src/lib/components/consignments/__stories__/consignments-bottom-aligned.story.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from "react"
+import { Text, View } from "react-native"
+
+import Search from "../components/artist-search-results"
+import BottomAlignedButton, { BottomAlignedProps } from "../components/bottom-aligned-button"
+
+export const name = "Consignments - bottom aligned"
+export const component = BottomAlignedButton
```

Comment:
These are used by the story and test generator to provide the names, and which component we're working with.